README.md

```diff
@@ -2,58 +2,57 @@
 
 Telegram bot for chats administration.
 
+[![Build Status](https://dev.azure.com/GrinderDotNetRu/Grinder/_apis/build/status/Keroosha.grinder?branchName=master)](https://dev.azure.com/GrinderDotNetRu/Grinder/_build/latest?definitionId=1&branchName=master)
+
 ## Configuration and build
 
-To build run you have to add a `bot_config.json` file on path `src\Grinder\`
+Just open project with Rider/VS 2019 and run build
 
-File structure:
+## Running
 
-```
-{
-  "Token": ...,
-  "ChannelId": ...,
-  "AdminUserId": ...,
-  "Socks5Proxy": {
-    "Hostname": "...",
-    "Port": ...,
-    "Username": "...",
-    "Password": "..."
-  },
-  "ChatsToMonitor": [
-    "chat @username",
-    ...
-  ],
-  "AllowedUsers": [
-    "username without at sign",
-    ...
-  ]
-}
-```
+### Using a Docker container
 
-`Token` - bot api token.
+You can just spawn new container with bot from: ***keroosha/grinder*** repo
 
-`ChannelId` - telegram channel id where bot whill write command logs.
+**(DO NOT FORGET ABOUT ENVIRONMENT VARIABLES)**
 
-`AdminUserId` - telegram user id who will be able to send private messages to bot.
+```bash
+docker run -d keroosha/grinder
+```
 
-`Socks5Proxy` - Socks5 proxy configuration.
+Or you can use already configured ***docker-compose.yml*** but, some action required before starting
 
-`ChatsToMonitor` - chats where bot will read messages and replyes.
+1. Create a ***config.env*** file with environments
+2. Change data path in ***docker-compose.yml***
+3. Setup database
+4. You are ready to go! ᕕ( ᐛ )ᕗ
 
-`AllowedUsers` - users, who will be able to execute commands in chats from `ChatsToMonitor`
+### Environment Variables
 
-### Build
+```ini
+# Socks5Proxy - Socks5 proxy configuration
+Grinder_Bot__Socks5Proxy__Port=1337
+Grinder_Bot__Socks5Proxy__Hostname=Hostme
+Grinder_Bot__Socks5Proxy__Password=Secrete
+Grinder_Bot__Socks5Proxy__Username=User
 
-To build run in docker on Raspberry Pi. From root directory:
+# AllowedUsers - users, who will be able to execute commands in chats from ChatsToMonitor
+Grinder_Bot__AllowedUsers__0=Pasha
+Grinder_Bot__AllowedUsers__1=Technique
```

Comment:
Honestly, these `__0` `__1` are terrible (the example itself is nice though!). I __can't believe__ we all've agreed that this is the best default option for settings.

Reply:
Alternative way via mounting appsettings.json are available + examples included

Reply:
Good.